mavutil.py

```diff
@@ -2458,6 +2458,20 @@ def __init__(self, offset, value, name):
             ret.append( EnumBitInfo(i, False, enum_entry_name) )
     return ret
 
+'''lookup table to map from a raw unit into a divisor and output unit'''
+dump_message_unit_decoder = {
+    "d%":     [10.0,       "%"],
```

Comment:
This could probably be neater as a tuple e.g.
```suggestion
    "d%":     (10.0,       "%"),
```

 - when pulling data out of the table you can assign into a tuple e.g. `(divisor, units) = dump_message_unit_decoder(x)`